app/src/main/kotlin/com/wire/android/ui/calling/initiating/InitiatingCallScreen.kt

```diff
@@ -52,15 +58,16 @@ fun InitiatingCallScreen(
 @OptIn(ExperimentalMaterialApi::class)
 @Composable
 private fun InitiatingCallContent(
+    errorState: InitiatingCallState.InitiatingCallError,
+    onErrorStartingCall: () -> Unit,
     callState: CallState,
     toggleMute: () -> Unit,
     toggleSpeaker: () -> Unit,
     toggleVideo: () -> Unit,
     onHangUpCall: () -> Unit,
     onVideoPreviewCreated: (view: View) -> Unit,
-    onSelfClearVideoPreview: () -> Unit
+    onSelfClearVideoPreview: () -> Unit,
```

Comment:
we don't need this comma  💁‍♂️

Reply:
I think we never agreed on dangling commas, but it's not that bad. It may look a bit uglier now, but it helps in the future when we add more items to the list:

![grafik](https://user-images.githubusercontent.com/9389043/186368694-ae9bed8e-111e-4554-8b6b-b72d32a9d2de.png)

Credits: https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8